src/femat.jl

```diff
@@ -83,5 +83,3 @@ LinearAlgebra.mul!(C, adjA::Adjoint{T,<:FeMat{T}}, B::FeMat{T}) where {T} =
 Does `A` have full column rank?
 """
 isfullrank(A::FeMat) = A.rank == length(A.piv)
-
-nθ(A::FeMat) = 0
```

Comment:
Not arguing for keeping this around, but why remove it? (Or rather what was the original use case that this was included? `allterms` stuff?)

Reply:
I guess there is no problem with keeping it around.  You are correct that it was added in the conversion to using `allterms` so that nθ of a model could be written as the sum of nθ applied to the `allterms` field.

My feeling is that it is better to avoid creating specific methods if we know we won't use them because deleting it cuts down on clutter and on those reading the code thinking "why is this here?".  It would probably be possible to drop all the methods for nθ but I haven't checked.

Reply:
Let's go ahead and cut it -- I think we're slowly converging on a somewhat more refined internal structure and trimming methods makes it easier to keep an overview of everything.